frontend/src/routes/stories/[story_id]/pages/edit/[id].svelte

```diff
@@ -0,0 +1,53 @@
+<script context="module">
+  export async function load({ params }) {
+    const { userId, storyId, pageId } = params;
+
+    let pageData;
+    if (pageId !== 'new') {
```

Comment:
Small comment of preference, when I see "ID" I think of numbers, not words like "new". Maybe it could be something like `if (pageIsNew)` which can hold a bool or a string `if (pageType === 'new;)` something like that might be clearer?

Reply:
Makes sense! I will fix this!